BepInEx/Utility.cs

```diff
@@ -237,5 +237,10 @@ public static bool TryOpenFileStream(string path, FileMode mode, out FileStream
 				return false;
 			}
 		}
+
+		internal static bool IsZero(this Version v)
```

Comment:
Considering that this is used in one place and I think the use case for this is really limited, it might be a good idea to just have this code explicitly written in the only call, and the utility method removed